portality/formcontext/emails.py

```diff
@@ -85,8 +85,41 @@ def send_assoc_editor_email(obj):
                         group_name=eg.name,
                         url_root=url_root)
 
+def send_publisher_update_request_editor_assigned_email(application):
+    """ Send email to publisher informing them an editor has been assigned """
+
+    owner = models.Account.pull(application.owner)
+    send_list = [
+        {
+            "name" : owner.name,
+            "email" : owner.email,
+            "sent_alert" : Messages.SENT_PUBLISHER_ASSIGNED_EMAIL,
+            "not_sent_alert" : Messages.NOT_SENT_PUBLISHER_ASSIGNED_EMAIL
+        }
+    ]
+
+    fro = app.config.get('SYSTEM_EMAIL_FROM', 'feedback@doaj.org')
+    subject = app.config.get("SERVICE_NAME","") + " - your application has been assigned an editor for review"
```

Comment:
this says application but the method is for an update request.

Reply:
I'll prepare a hotfix for this.